### ISSUE TITLE
This came with a bunch of bloatware, what gives?

### ISSUE THREAD

**Issue by JadeShrineMaiden** [cheat-engine/cheat-engine#2027]
I got this to test my own game's anti-cheat functions, it installed a bunch of bloatware, had to wipe my pc clean... do i really need to compile this from scratch to not deal with this bs?

**Comment by Measurity** [comment]
No, just follow the instructions of the installer to skip the adware.

**Comment by fade2gray** [comment]
@JadeShrineMaiden 
![SkipAll](https://user-images.githubusercontent.com/1332009/151723127-37a7d4c3-bc6c-4a0c-ba65-164e3df2d759.png)


Watch the video #1954

**Comment by VioletFalcon** [comment]
The binary provided here is just a downloader for the real installer binary, shady practice.
And also comes with adware.
Such a shame. If someone forked this to remove the trash please tell me.

**Comment by fade2gray** [comment]
@VioletFalcon Fork and compile your own, problem solved.

**Comment by VioletFalcon** [comment]
> @VioletFalcon Fork and compile your own, problem solved.

Yes, thanks

**Comment by protosam** [comment]
> The binary provided here is just a downloader for the real installer binary, shady practice.

It's not just shady, but illegal.

Catalogging this thread for ref https://github.com/cheat-engine/cheat-engine/issues/2105#issuecomment-1188365721
